src/doom/p_mobj.c

```diff
@@ -727,16 +727,16 @@ void P_RespawnSpecials (void)
     int			i;
 
     // only respawn items in deathmatch
-    if (deathmatch != 2)
-	return;	// 
+    if (deathmatch != 2 && deathmatch != 3)
```

Comment:
Please leave a comment here indicating that `deathmatch != 3` is a Crispy-specific change.